Automattic/vip-design-system
# Convert the Accordion to Typescript

## Description

Converted the Accordion component to typescript. 

## Checklist


## Steps to Test


1. Pull down PR.
1. `npm run dev`.
1. Open Storybook.
1. Check the accordion is working as it used to
